vtex-apps/checkout-resources
# Add general messages

_This PR depends on and must be merged after https://github.com/vtex/checkout-graphql/pull/19._

#### What problem is this solving?

This PR adds the new `generalMessages` field to the queries/mutations made by Checkout IO. Those will be displayed in the UI in a general context.

#### How should this be manually tested?

Use [this workspace](https://general--vtexgame1.myvtex.com), add some item, go to `/cart`, set the quantity to some number greater than 50 and verify a message is displayed.

#### Checklist/Reminders

- [x] [Linked this PR to a Clubhouse story (if applicable)](https://app.clubhouse.io/vtex/story/20914/implementar-toasts).

#### Screenshots or example usage

#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes
